### PR TITLE
Sets character limit on shot title (WIP)

### PR DESCRIPTION
**Pull request by niharikak101**
FIx for #2147

### CODE REVIEW

**Comment by dannycoates** [comment]
This isn't really my area of expertise but might we use [text-overflow](https://developer.mozilla.org/en/docs/Web/CSS/text-overflow) CSS here instead?

**Comment by ianb** [comment]
I think we have it working in server/src/pages/shot/view.js through truncating (doesn't look great, but it's better than when #2147 was opened).  But I think the issue with My Shots still remains.  

@niharikak101: can you change this to only due the truncation on My Shots?

As written `displayTitle(title)` is also duplicated across shot/view.js and shotindex/view.js though if this is removed from shot/view.js then that won't matter.

File: server/src/pages/shotindex/view.js [hunk 1]
```diff
@@ -130,6 +130,15 @@ class Body extends React.Component {
     );
   }
 
+  displayTitle(title) {
+    if(title.length > 140) {
```
Comment (ianb) [inline server/src/pages/shotindex/view.js]: There should be a space after "if"

File: server/src/pages/shotindex/view.js [hunk 2]
```diff
@@ -130,6 +130,15 @@ class Body extends React.Component {
...
+      return (title.substring(0, 140) + "...");
+    }
+    else {
```
Comment (ianb) [inline server/src/pages/shotindex/view.js]: "else" should be on the same line as "}", like: `} else {`

**Comment by niharikak101** [comment]
@ianb @dannycoates 
Understood :) I'll make the changes!

**Comment by ianb** [comment]
Thanks!

**Pull request merged**